rxnetty-common/src/main/java/io/reactivex/netty/channel/BackpressureManagingHandler.java

```diff
@@ -411,6 +448,12 @@ private void requestMoreIfWritable(Channel channel) {
 
         private final ChannelHandlerContext ctx;
         private final ChannelPromise overarchingWritePromise;
+
+        private final int initialRequest;
+        private long maxBufferSize;
+        private long pending; /*Guarded by guard*/
+        private long lowWaterMark;
+
         private final Object guard = new Object();
```

Comment:
@stevegury yes you are correct, I would change this in a separate PR as the change isn't related.